core/src/wallet/cosmos/api_impl/CosmosLikeTransactionApi.cpp

```diff
@@ -340,5 +340,99 @@ namespace ledger {
             return *this;
         }
 
+        std::string CosmosLikeTransactionApi::serializeTransfer() const {
+            auto document = rapidjson::Document();
+            document.SetObject();
+            auto& allocator = document.GetAllocator();
+
+            auto baseReq = rapidjson::Value(rapidjson::kObjectType);
+            for (const auto &message : _txData.messages) {
+                const auto tmpMessage = std::make_shared<CosmosLikeMessage>(message);
+                if (!(tmpMessage->getMessageType() == api::CosmosLikeMsgType::MSGSEND)) continue;
```

Comment:
This means currently the gas estimation will only serialize transfers to send the simulate request ?

Reply:
I forgot to remove this function for CosmosLikeTransactonApi. It has been moved to the Gaia explorer with the functions to serialize other messages (delegation, rewards, undelegation, redelegation). I will remove it.